CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# Upcoming Release
```

Comment:
Please mention that the underlying syscall used for installing the filter changed from `prctl` to `seccomp`.
I don't think this will be a breaking change in any way, but it's good to have in mind.